lang/swi-prolog-devel/Portfile

```diff
@@ -1,64 +1,65 @@
-PortSystem    1.0
-PortGroup     cmake 1.1
+PortSystem     1.0
+PortGroup      cmake 1.1
 
-name		swi-prolog-devel
-conflicts	swi-prolog swi-prolog-lite
-epoch		20051223
-version		8.1.2
+name           swi-prolog-devel
+conflicts      swi-prolog swi-prolog-lite
+epoch          20051223
+version        8.1.3
 
-categories	lang
-license		LGPL-2.1
-maintainers	{uva.nl:J.Wielemaker @JanWielemaker} {logtalk.org:pmoura @pmoura}
-platforms	darwin
-description	SWI-Prolog compiler plus extra packages (development version)
+categories     lang
+license        BSD
```

Comment:
You changed the license and did not mention why. Please update the commit message describing the reason for this change. Was it wrong before or did it suddenly change this version?

Reply:
SWI-Prolog moved to BSD recently. All 8.x versions are under the new license.